frontend/src/components/core/StreamlitDialog/StreamlitDialog.test.tsx

```diff
@@ -73,3 +75,53 @@ describe("StreamlitDialog", () => {
     expect(wrapper.find("StyledTertiaryButton")).toMatchSnapshot()
   })
 })
+
+describe("aboutDialog", () => {
+  beforeEach(() => {
+    SessionInfo.current = new SessionInfo({
+      appId: "aid",
+      sessionId: "sessionId",
+      streamlitVersion: "42.42.42",
+      pythonVersion: "pv",
+      installationId: "iid",
+      installationIdV3: "iid3",
+      authorEmail: "ae",
+      maxCachedMessageAge: 2,
+      commandLine: "command line",
+      userMapboxToken: "mpt",
+    } as SessionInfoArgs)
+  })
+
+  afterEach(() => {
+    const UnsafeSessionInfo = SessionInfo as any
+    UnsafeSessionInfo.singleton = undefined
+  })
+
+  it("renders about dialog with Streamlit version included", async () => {
```

Comment:
Nit: slightly confusing test description. How about `it("shows version string if SessionInfo exists")`? And the one below could be `it("shows no version string if SessionInfo does not exist")`